src/plugins/GUITestBase/src/GTUtilsProjectTreeView.cpp

```diff
@@ -49,6 +49,16 @@ using namespace HI;
 
 const QString GTUtilsProjectTreeView::widgetName = "documentTreeWidget";
 
+#define GT_METHOD_NAME "countFilesInProjectView"
+int GTUtilsProjectTreeView::countFilesInProjectView(HI::GUITestOpStatus& os) {
```

Comment:
The method is incorrect because project view may be in the filtering state.
You can just rename it to "countTopLevelItems" - in this case it will be good.